cachito/web/models.py

```diff
@@ -197,15 +222,19 @@ def to_json(self, verbose=True):
         if verbose:
             rv.update({'state_history': states})
             rv.update({'dependencies': [dep.to_json() for dep in self.dependencies]})
+            rv.update(
+                {'dependency_overrides': [dep.to_json() for dep in self.dependency_overrides]}
```

Comment:
Hmm... does this mean the `dependencies` attribute is not an accurate representation of the dependencies being used?

Reply:
It would be more useful if the existing items in the `dependencies` list had additional information to determine whether or not they are overrides. That way clients only have to consume a single list, but still have the overrides information if needed.

Reply:
The `dependencies` attribute is accurate. It's the dependencies that golang will use during a build, which includes the replace directives in the `go.mod` file.

I'll reach out on IRC on how we want to approach the comment on the JSON representation.